slasher/detection/attestations/mock_spanner.go

```diff
@@ -20,20 +20,27 @@ type MockSpanDetector struct {
 	lock  sync.RWMutex
 }
 
-// DetectSlashingForValidator uses a validator index and its corresponding
-// min-max spans during an epoch to detect an epoch in which the validator
-// committed a slashable attestation.
+// DetectSlashingForValidator mocks a detected slashing, if the sent attestation data
+// has a source epoch of 0, nothing will be detected. If the sent attestation data has a target
+// epoch equal to or greater than 6, it will "detect" a surrounded vote for the target epoch + 1.
+// If the target epoch is greater than 12, it will "detect" a surrounding vote for target epoch - 1.
+// Lastly, if it has a target epoch less than 6, it will "detect" a double vote for the target epoch.
 func (s *MockSpanDetector) DetectSlashingForValidator(
 	ctx context.Context,
 	validatorIdx uint64,
 	attData *ethpb.AttestationData,
 ) (*types.DetectionResult, error) {
-	if attData.Target.Epoch == 0 {
```

Comment:
this can probably be written more cleanly, we can use a select with the relevant cases: Ex: 
`attData.Target.Epoch > 12 `
`attData.Source.Epoch == 0`
etc. Instead of this long continous if-else